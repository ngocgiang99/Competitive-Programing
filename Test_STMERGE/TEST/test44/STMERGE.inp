10
4 2
141317512 966854894 
138372459 773697401 
684250874 237414673 
989906486 108862602 
7 4
698577560 519995780 27110934 516684611 
554465013 861204573 437787275 253539644 
849757683 30161227 273175691 560658299 
795171958 145004737 517550875 687305877 
196025252 410670965 246795136 958779232 
873226978 126786652 14218560 760949488 
743959508 723633323 400657950 878559387 
2 10
357372449 176967157 679664299 52796820 428475789 755085523 3734809 956056024 936755431 941524811 
191765072 112688857 258782452 814941617 764091701 729212682 93570745 953456283 177661676 737829665 
7 3
359068891 750753811 20679872 
330546060 534292134 321670933 
355881036 874790910 738644019 
120696732 819847203 412876231 
972327847 881570140 92242572 
531229884 136044390 659533900 
798998027 219502835 875528726 
3 6
681963713 33150108 771958476 62813088 310497977 512048156 
796902278 781528064 444974514 848581628 168823235 181732361 
357304001 461819361 943606456 125022082 770405178 78079407 
2 1
522917554 
862026636 
10 7
764849407 935850973 223669177 979237369 919930933 235772230 272550559 
679561697 191754409 488585900 780964359 362348496 675574001 821640502 
584031908 181277453 451412961 964417607 524192620 581897499 165019279 
317675909 398065045 546998721 556254287 692484820 480733330 421439747 
602934966 680862810 332030489 354795815 976521626 429410940 538393118 
95100080 268452111 309630430 811258072 251867236 126649194 869219686 
688817339 767440201 635504961 961624120 31740027 832765091 108136809 
329363387 900246799 368592887 881083637 542633361 242988970 334210826 
671642429 723921314 552799170 420342241 849975763 349671953 692373772 
826866881 872284302 225647428 547236733 9189424 946660226 376495071 
6 3
551340293 148564680 140668648 
64295435 724368299 23905572 
595136779 946997442 268667465 
405022295 878548138 415201947 
200689141 685471239 73374968 
167148379 43832835 763627545 
9 1
966632457 
287830705 
240219065 
972474853 
389917147 
884713013 
56269269 
164484728 
11650446 
10 9
98586503 137170296 745997524 475627150 477387798 721943597 958410745 111970388 25301866 
456994947 143777821 616190526 486867720 19816334 653590758 267673472 77655221 951146147 
697973305 841594766 715910775 463395160 616401466 16426526 338093201 871031681 326717775 
752408357 967671178 92273460 663033885 552146137 531164710 259985605 273853832 29618605 
568286570 745502409 89710262 322099798 671739547 982657046 284516108 14180299 71450430 
465922496 750432814 25041216 192051993 408087324 318358530 485133865 96999471 655770048 
618155692 325671997 314220231 659962240 266698386 943247727 959221093 393140850 412961369 
834072053 136168142 625725674 607559398 511383109 635239943 490204029 359671133 319161106 
121068117 369545473 341321507 78715217 467231038 873022091 290018783 372903993 170126469 
159839086 408320884 350162238 171750449 265221441 94711815 440886804 464948358 150997977 
