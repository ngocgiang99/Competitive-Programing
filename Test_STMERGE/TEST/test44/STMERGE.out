250180114
516684611
191765072
20679872
309715730
522917554
203236889
140668648
11650446
25301866
